tool.go

```diff
@@ -498,7 +498,21 @@ func main() {
 	cmdServe.Run = func(cmd *cobra.Command, args []string) {
 		options.BuildTags = strings.Fields(*tags)
 		dirs := append(filepath.SplitList(build.Default.GOPATH), build.Default.GOROOT)
-		sourceFiles := http.FileServer(serveCommandFileSystem{options: options, dirs: dirs, sourceMaps: make(map[string][]byte)})
+		root := ""
```

Comment:
I can point you to [this article](http://dave.cheney.net/2014/05/24/on-declaring-variables) by Dave Cheney that lists some suggestions for variable declaration styles and when to use them. I think they're good suggestions and they match well most of the idiomatic Go code I see.

Both forms are okay, but I think `var root string` is slightly nicer. It emphasizes that you're just declaring a variable and not initializing it with an important value.

You can see examples of this style in the same file, e.g., [here](https://github.com/gopherjs/gopherjs/blob/2fecf3fff15b993f5ea9c095868ef1ac0ba6a9a8/tool.go#L404).